packages/material-ui/src/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -1,34 +1,21 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import {
-  getClasses,
-  createMount,
-  describeConformance,
-  act,
-  createClientRender,
-  screen,
-} from 'test/utils';
+import { createMount, describeConformanceV5, act, createClientRender, screen } from 'test/utils';
 import Breadcrumbs from './Breadcrumbs';
+import classes from './breadcrumbsClasses';
 
 describe('<Breadcrumbs />', () => {
   const mount = createMount();
-  let classes;
   const render = createClientRender();
 
-  before(() => {
-    classes = getClasses(
-      <Breadcrumbs>
-        <span>Hello World</span>
-      </Breadcrumbs>,
-    );
-  });
-
-  describeConformance(<Breadcrumbs>Conformance?</Breadcrumbs>, () => ({
+  describeConformanceV5(<Breadcrumbs>Conformance?</Breadcrumbs>, () => ({
     classes,
     inheritComponent: 'nav',
     mount,
+    muiName: 'MuiBreadcrumbs',
     refInstanceof: window.HTMLElement,
     testComponentPropWith: 'div',
+    skip: ['componentsProp', 'themeVariants', 'themeDefaultProps'],
```

Comment:
Let's not skip the `themeVariants` and `themeDefaultProps`.

Reply:
I'm realizing that I was limited in my thinking of the scope of "variants", and for some reason I was thinking that it was just meaning the `variant` prop...don't know why.  Fixing!